src/swarmus_pioneer/launch/pioneer_bringup.launch

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?>
+<launch>
+    <!-- Start nodes for peripherals -->
+    <include file="$(find swarmus_pioneer)/launch/pioneer_peripherals.launch"/>
+</launch>
```

Comment:
Have been looking at other packages to have A more clear idea about with is a _bringup_ launch file. From what I saw, the bringup goal is to start all the sensors and actuator driver of the real robot. So I think that the pioneer_peripherals.launch is not needed and its content should be directly in the _pioneer_bringup.launch_.

Reply:
Do the bringups also contain the software/application for the robot? I was under this impression, so that is why I separated a peripheral portion so we could have separated peripherals, Swarm and applicative launch files.

Reply:
I wasn't able to find a reference document on what should the standard bringup do, but from 3 examples (racecar-UdeS, turtlebot and Secrubot) I've notice that the bringup is only used to launch the drivers. In addition to that, the bringup is robot-specific where the applicative code should be mostly independent from the robot, so it should have its own package.

So I think we should only used the bringup like I suggested and when we will have the applicative code, we could add another launchfile that calls the bringup and the applicative code. In the end, we will achieve the same thing its just a question of naming standard.